src/templates/components/imageAndStaticMap/index.tsx

```diff
@@ -14,18 +13,11 @@ export const ImageAndStaticMap: React.FC<ImageAndStaticMapProps> = ({
 }) => {
   return (
     <div className="usa-width-one-third inline-table-helper vads-u-order--first mobile-lg:vads-u-order--initial vads-u-margin-bottom--2 vads-u-margin-left--auto facility">
-      <div
-        onClick={() =>
-          recordEvent({ event: 'image-click', 'facility-name': facilityId })
-        }
-        style={{ cursor: 'pointer' }}
-      >
-        <MediaImage
-          {...image}
-          imageStyle="2_1_large"
-          className="facility-img"
-        />
-      </div>
+      <MediaImage
+        {...image}
+        imageStyle="3_2_medium_thumbnail"
```

Comment:
Changing this from `2_1_large` to `3_2_thumbnail` will also affect vet centers and vet center outstations. @laflannery can you double-check that those are still okay? (I'm not sure how to get to those pages right now. 🙈 )

If we need, we can always prop thread this to change it up per content type that needs to display this image and map combo.

Reply:
@cvalarida All facility images should have the same ratio, we have very similar layout for these facility. I confirmed Vet center facility specifically though, [here's an example](https://www.va.gov/boston-vet-center/)) but there are no Outstations because these were never finished. However, Outstations were intended to essentially be a copy of the main Vet center pages so those images should be the same as well.